Ignore detrimental values

karelbilek opened the pull request:
This should be added to all "accumulative" wallets, but at this point, I don't care really; since the code shoule be refactored anyway, to avoid all this copy-pasting

And in the end I don't want to refactor it :D

karelbilek committed:
Ignore detrimental values

This should be added to all "accumulative" wallets, but at this point, I don't care really; since the code shoule be refactored anyway, to avoid all this copy-pasting

And in the end I don't want to refactor it :D

runn1ng commented:
I am adding all these PRs which are incompatible with one another :/ sorry, don't know how to do it well

karelbilek committed:
Exiting infinite loop

karelbilek committed:
Actually, skip too small for input

It will get detrimental to the *other user*

Also it's a dust threshold in bitcoin itself so it won't get posted anyway
